### ISSUE TITLE
Custom skin not staying

### ISSUE THREAD

**Issue by MDrapper** [minecraft-linux/mcpelauncher-manifest#684]
**Describe the bug**
Custom skin not staying when I add a custom skin and then restart the game it puts my character back to Steve

**To Reproduce**
Add a custom skin restart the game and see if you still have a custom skin

**Expected behavior**
Should've left the skin on the character when I restart the game

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [macOS]
 - Version [v0.3.4-macOS-x86_64-0.2.361 (build 361))]
 - Installation Type [IDK]

**Comment by ghost** [comment]
Skins is not saving and loading. Skin storage is broken after version 1.16.200.

**Comment by Nunnito** [comment]
Same here. I'm using Arch Linux with the latest Appimage

**Comment by p0ryae** [comment]
yeah no skins load/save at all on my end. I gotta create a new character each time. I launch into the game

**Comment by Offroaders123** [comment]
I have the same issue here. I also noticed this is the same case for any custom skin packs I have made myself, like ones from outside the marketplace. The packs stay in the game as expected, but using a skin from them doesn't persist to the next time I open the game again.

**Comment by wertie3020** [comment]
i have a solution chage the basic skins for Steve and alex by your own skin
open game data folder
then go to versions/1.18.12.01/assets/skin_packs/vanilla
alex.png and steve.png

**Comment by VoxelTek** [comment]
Still present on launcher v1.2.0, Minecraft version 1.21.71.01, on x86_64 Ubuntu 24.04.

However, the issue appears to be somewhat different, as the skin is kept, but not applied.

https://github.com/user-attachments/assets/9210723c-2d48-4be5-94a6-f44ef3fd398a

**Comment by VoxelTek** [comment]
UPDATE: this was solved by re-adding the skin.
